package.json

```diff
@@ -37,9 +36,9 @@
     "homepage": "https://github.com/humphd/next#readme",
     "dependencies": {
         "dexie": "^2.0.4",
-        "filer": "humphd/filer#9p-filer",
+        "filer": "humphd/filer#a3bfd5d",
```

Comment:
I don't think this is a good idea because every time there is a commit on either branch, we will need to update this. Should't there be a way to clear Travis' cache?

Reply:
https://travis-ci.org/humphd/next/jobs/395951218

In this case, the `git sha` it wants to use no longer exists (I rebased) and it's completely lost.  I tried removing caches from Travis, no effect.